framework/tests/org.jboss.tools.rsp.secure.test/src/org/jboss/tools/rsp/secure/test/model/RSPSecureStorageTest.java

```diff
@@ -0,0 +1,173 @@
+package org.jboss.tools.rsp.secure.test.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.jboss.tools.rsp.secure.crypto.CryptoException;
+import org.jboss.tools.rsp.secure.model.ISecureStorage.ISecureNode;
+import org.jboss.tools.rsp.secure.model.RSPSecureStorage;
+import org.jboss.tools.rsp.secure.model.RSPSecureStorage;
+import org.jboss.tools.rsp.secure.model.RSPSecureStorage.SecureNode;
+import org.jboss.tools.rsp.secure.model.ISecureStorage.ISecureNode;
+import org.jboss.tools.rsp.secure.model.RSPSecureStorage;
+import org.junit.Test;
+
+public class RSPSecureStorageTest {
+	@Test
+	public void testNonExistentProperties() throws IOException, CryptoException {
```

Comment:
lengthy test methods are better off when torn appart where each block is in it's own method. This helps identifying the exact failure

Reply:
not a deal-braker. Can live with you disagreeing here. resolving.